torch/csrc/autograd/profiler.cpp

```diff
@@ -340,3 +347,45 @@ void RecordProfile::processEvents(const std::vector<Event*>& events) {
 }
 
 }}}
+
+
+namespace {
+
+void callBoxedWorkaround(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  // This should just be op.callBoxed(stack), but that doesn't work for all ops yet.
+  // Note: If op.callBoxed(stack) works for you, then that is preferrable because
+  // it's much faster and doesn't come with a dependency on JIT code.
+  // Instead, we take a path through the JIT operator registry, which has a boxed
+  // calling mechanism that works for all ops from native_functions.yaml.
+
+  auto s = c10::Symbol::fromQualString(op.schema().name());
```

Comment:
warning: this re-dispatch is probably slow. But it's the only way that works for 100% of ops currently.

Reply:
At the very least, we can speed up the process of looking up the JIT operator, if we introduce some sort of mapping from OperatorHandle to JIT schema. @smessmer do you know if this can be conveniently done?

Reply:
@smessmer  We need to make a call on what should be happening here. @jamesr66a do you think you could characterize when the boxed workaround gets called? It's going to just be for custom operators which didn't have VariableType generated for them. How were these supported before?

I just really don't think it's acceptable to land a silent slowdown that will affect all custom operators when you're benchmarking. It would be better to just not have accurate profiling information in this case at all.

Reply:
we are relatively close to making the non-workaround callBoxed work for redispatching, but it'll likely take a few more days

Reply:
How about for now, I remove this and reinstate the `RECORD_FUNCTION` at https://github.com/pytorch/pytorch/pull/33057/files#diff-94205457152f6821c30329a4ddb40bbeL21, and then we can add this back with the proper callBoxed call later?